.solcover.js

```diff
@@ -1,3 +1,5 @@
 module.exports = {
-  skipFiles: ["test/"],
+  // TODO(nlordell): Stop skipping coverage for the vault relayer once it starts
+  // actually doing stuff.
+  skipFiles: ["test/", "src/contracts/GPv2VaultRelayer.sol"],
```

Comment:
I'd say it makes sense to merge this PR with a decrease coverage and then see coverage go back up again, so we don't forget to undo this change.